docs/events/timeline-ui-overview.asciidoc

```diff
@@ -178,3 +178,29 @@ From the *Correlation* tab, you can also do the following:
 * Specify the date and time range that you want to investigate.
 * Reorder the columns and choose which fields to display.
 * Choose a data view and whether to show detection alerts only.
+
+[discrete]
+[[esql-in-timeline]]
+== Use {esql} to interact with alerts 
```

Comment:
Changing "alerts" to "events" or "documents" throughout this section as per @paulewing's request via slack.

```suggestion
== Use {esql} to interact with events 
```